enhancements/proposals/node-autoreplacement.md

```diff
@@ -0,0 +1,133 @@
+# Node Autoreplacement
+
+## Summary
+
+Node Autoreplacement enhancement extends Scylla Operator with a functionality of automated replacement of nodes which lost their storage, which, at this point, renders them getting stuck on joining the cluster.
+
+This proposal suggests creating a module responsible for triggering a node replacement in such circumstances.
+As a side effect, introducing such functionality would allow for simplifying the existing `OrphanedPVController` by
+reducing its task to evicting orphaned PVs.
+
+## Motivation
+
+Currently, when a node looses its storage, it gets stuck on joining a cluster, since Scylla produces a following runtime error:
+```
+Startup failed: std::runtime_error (A node with address <ip> already exists, cancelling join. Use replace_address if you want to replace this node.)
+```
+
+Successfully restarting the node requires a manual intervention of a human operator by annotating the
+service corresponding to the affected node with a `scylla/replace` label. The main goal of this proposal is to improve
+Scylla Operator's stability in running Scylla independently of its administrators' interventions.
+
+### Goals
+
+- Successfully discovering and autoreplacing nodes which lost their storage without requiring user intervention.
+
+### Non-Goals
+
+- Removing orphaned PVs. This is left for `OrphanedPVController`.
+
+## Proposal
```

Comment:
explicitly mention what happens to a scyllacluster with 1 node (already bootstrapped)